docs/user/reference/general/register-custom-pings.md

```diff
@@ -77,8 +77,28 @@ pings = load_pings(resource_filename(__name__, "pings.yaml"))
 
 </div>
 
-<div data-lang="Rust" class="tab"></div>
+<div data-lang="Rust" class="tab">
+
+In Rust custom pings need to be registered individually.
+This should be done before calling `glean::initialize`.
+
+```Rust
+use your_glean_metrics::pings;
+
+glean::register_ping_type(&pings::custom_ping);
+glean::initialize(cfg, client_info);
+```
+
+</div>
 <div data-lang="Javascript" class="tab"></div>
-<div data-lang="Firefox Desktop" class="tab"></div>
+<div data-lang="Firefox Desktop" class="tab">
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+##### No custom ping registration needed
+
+> On Firefox Desktop all custom pings are registered automatically.
```

Comment:
nit: This information could be presented as a [`data-info`](http://localhost:3000/appendix/contribution-guidelines.html#data-info) tooltip instead of requiring the tab to be opened :)